### PR TITLE
Drop the need to patch the virtualenv activation script

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
* create a dummy setup.py that can be used for editable install of the project folder
  (removes the need to patch PYTHONPATH)
* disable bytecode in conftest for current process and subprocesses
  (removes the need to set PYTHONDONTWRITEBYTECODE)

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 591eed5f50c82dcfaf33579520b271f5a206ad05:

Everything seems all right :smile: .
*CFME QE Bot*

File: docs/getting_started.rst [hunk 1]
```diff
@@ -55,26 +53,7 @@ Detailed steps (manual environment setup):
...
-* If you forgot to do that and you have ``pyc`` files polluting your folders, this is a cure, placed
-  in ``~/.bashrc`` and usable everywhere:
-
-  .. code-block:: bash
-
-    alias rmpyc='find . -name \*.pyc -delete; find . -name __pycache__ -delete'
-
-  Reload your ``.bashrc`` and issue ``rmpyc`` command.
-
+* Fork and Clone this repository.d
```
Comment (psav) [inline docs/getting_started.rst]: type (.d)

**Comment by RonnyPfannschmidt** [comment]
dont merge yet

**Comment by RonnyPfannschmidt** [comment]
i missed a bit in artifactor where it sets pythonpath at subprocess startup ^^

**Comment by dajoRH** [comment]
Lint report for commit d1a1a370a374c3b18aafa87652e7ac19f9020af9:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b3276d2be21a15c8dc2c1bc407a802d03d953a71:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ec62399f4d17fa1b0dad97d93458cfb6a2708342:

`scripts/disable-bytecode.py`:
- :grey_exclamation: Line 13:20: **S001** *found %r formatter*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b0c21385506e66edf9160763b9c6454749d1f680:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**